scripts/apps/authoring/styles/authoring.scss

```diff
@@ -212,99 +212,150 @@
+            }
+            &:active {
+                opacity: 1;
+                background-color: rgba(255, 255, 255, 0.12);
+            }
         }
-        .title {
-            flex-grow: 1;
-            display: block;
+        .opened-articles-bar__item-title {
+            display: flex;
+            flex: 1 1;
             overflow: hidden;
             text-overflow: ellipsis;
+            align-items: center;
             white-space: nowrap;
             color: #8bcaf7;
             font-size: 1.2rem;
+            transition: color 0.2s ease-in-out;
             @include text-light();
+            i {
+                margin-right: 0.4rem;
+            }
             &:hover {
-                color: #fff;
+                color: $white;
                 text-decoration: none;
                 cursor: pointer;
             }
         }
         &:active {
-            background: #444;
+            background: #333;
         }
-        &.active {
-            background: darken($sd-blue,10%) !important;
+        &.opened-articles-bar__item--active {
+            background: darken($sd-blue,18%);
+            border-color: $bottom-bar-item-border-color-active;
             &:hover {
                 cursor: default;
             }
-            .title {
-                color: #fff;
+            &:active {
+                background: darken($sd-blue,10%);
+            }
+            .opened-articles-bar__item-title {
+                color: $white;
+            }
+        }
+        &--teal { // use this outside the <ul>, as a single item
+            border-right: none;
+            border-left: 1px solid $bottom-bar-item-border-color;
+            margin-right: 1px;
+            &:hover {
+                background: rgba(0, 150, 136, 0.2);
+            }
+            .opened-articles-bar__item-title {
+                color: rgb(0, 242, 219);
+            }
+            &.opened-articles-bar__item--active, &.opened-articles-bar__item--active:active {
+                background: rgb(0, 150, 136);
+                color: $white;
             }
         }
     }
-    .list {
+    .opened-articles-bar__quick-actions {
+        grid-column: 1/2;
         display: flex;
-        flex-wrap: wrap;
-        margin-left: 31px;
-        margin-right: 250px;
-        &.full-width {
-            margin-right: 0;
-        }
-        > li {
-            flex: 1 1 auto;
-            border-right: 1px solid #333;
+        align-items: center;
+        justify-content: center;
+        background: rgba(0, 0, 0, 0.4);
+        border: 0;
+        height: 3.2rem;
+        width: 4.8rem;
+        text-align: center;
+        &:hover {
+            cursor: pointer;
+            background: $sd-blue;
         }
     }
-    .multiedit-item {
-        float: right;
-        > li {
-            border-left: 1px solid #333;
-            width: 24rem;
+    .opened-articles-bar__multiedit {
+        grid-column: 3/4;
```

Comment:
I'd be good if it were more flexible and adding/removing buttons didn't require stylesheet changes.

Reply:
It doesn't require changes.

Reply:
I was referring to [action buttons on the right](https://user-images.githubusercontent.com/2076953/76771875-49466580-67a0-11ea-839c-c8f32bbab656.png). If you add another, one gets pushed out. It'd be nice to have a container element where you can put as many buttons as you want and they simply expand the container to fit.

Reply:
I don't think this is necessary. This element does not have to be flexible that way. All changes that may come in the future should go trough the UI design team anyway and if some CSS adaptation is needed we will do them.

Reply:
Many things aren't necessary if you think about it this way. Using a variable for that 48px value isn't necessary either - worked as good without it. It's a matter of clean code to allow for flexibility even if you don't need it just yet, especially when it's not that hard to do...